modules/sso-jdk/artifacts/opt/jboss/container/openjdk/jdk/jvm-options

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# ==============================================================================
+# JDK specific customizations
+#
+# ==============================================================================
+
+# Remove the following function with OpenJ9 0.18.0 is available. See JBEAP-17984
+function jvm_specific_options() {
+    if [[ $(uname -i) = s390x ]] ; then
+        echo "-Xmx2g"
+    else
+        :
+    fi
+}
+
+function jvm_specific_diagnostics() {
+    if [[ $(uname -i) = s390x ]] ; then
+        echo ""
+    else
+        echo "-XX:NativeMemoryTracking=summary -XX:+PrintGC -XX:+PrintGCDateStamps -XX:+PrintGCTimeStamps -XX:+UnlockDiagnosticVMOptions"
```

Comment:
More OOC, than a real issue. This comes from https://github.com/jboss-openshift/cct_module/blob/master/jboss/container/openjdk/jdk/8/artifacts/opt/jboss/container/openjdk/jdk/jvm-options#L9  -- since we are the default `jboss.container.openjdk.jdk` module with the `sso-jdk` one, we need to add those back for x86_64 arch, right?

Reply:
exactly, kind of inheritance without actual chance for inheritance

Reply:
Ok, thanks for clarification